lib/pharos_cluster.rb

```diff
@@ -9,7 +9,7 @@
 
 module Pharos
   CRIO_VERSION = '1.10.6'
-  KUBE_VERSION = ENV.fetch('KUBE_VERSION') { '1.10.5' }
+  KUBE_VERSION = ENV.fetch('KUBE_VERSION') { '1.11.0' }
   KUBEADM_VERSION = ENV.fetch('KUBEADM_VERSION') { KUBE_VERSION }
   ETCD_VERSION = ENV.fetch('ETCD_VERSION') { '3.1.12' }
```

Comment:
Etcd version is still the same?

Reply:
Myes, seems to have gotten lost in all of the `git add -p` I had to do to test this with the external kube repos.

```
    	[WARNING ExternalEtcdVersion]: this version of kubeadm only supports external etcd version >= 3.2.17. Current version: 3.1.12
```